pkg/inject/config.go

```diff
@@ -112,7 +114,9 @@ type Config struct {
 	StatsDSocketPath     string
 	XRayImage            string
 
-	ClusterName string
+	ClusterName     string
+	UseFipsEndpoint int
+	UseDualStack    int
```

Comment:
These seem like they should be bool instead of int. That's also consistent with the AWS_USE_FIPS_ENDPOINT env var that pops up other places.